generators/app/templates/package.json

```diff
@@ -17,75 +18,43 @@
   },
+  "homepage": "https://github.com/cerner/terra-ui#readme",
   "devDependencies": {
-    "@kadira/storybook": "^2.21.0",
-    "@kadira/storybook-addon-knobs": "^1.7.1",
-    "@kadira/storybook-addon-options": "^1.0.1",
-    "autoprefixer": "^6.6.1",
-    "babel-cli": "^6.18.0",
-    "babel-jest": "^18.0.0",
-    "babel-plugin-transform-object-rest-spread": "^6.22.0",
-    "babel-preset-es2015": "^6.18.0",
-    "babel-preset-react": "^6.16.0",
-    "css-loader": "^0.26.1",
-    "enzyme": "^2.7.0",
-    "enzyme-to-json": "^1.4.5",
-    "eslint": "^3.13.1",
-    "eslint-config-airbnb": "^14.0.0",
-    "eslint-plugin-import": "^2.2.0",
-    "eslint-plugin-jsx-a11y": "^3.0.2",
-    "eslint-plugin-react": "^6.9.0",
-    "gh-pages": "^0.12.0",
-    "identity-obj-proxy": "^3.0.0",
-    "jest": "^18.1.0",
-    "node-sass": "^4.3.0",
-    "postcss-loader": "^1.2.2",
-    "react-addons-test-utils": "^15.4.2",
-    "react-storybook-addon-props-combinations": "^0.3.0",
-    "sass-loader": "^4.1.1",
-    "storybook-addon-a11y": "0.0.4",
-    "storybook-addon-i18n-tools": "^1.0.0",
-    "storyshots": "^3.2.0",
-    "style-loader": "^0.13.1",
-    "stylelint": "^7.7.1",
-    "stylelint-config-sass-guidelines": "^1.1.1",
-    "terra-application": "^0.2.0",
-    "terra-legacy-theme": "^0.1.0",
-    "webpack": "^1.14.0",
-    "webpack-dev-server": "^1.16.2"
+    "terra-toolkit": "^0.1.0"
+  },
+  "peerDependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2"
   },
   "dependencies": {
     "classnames": "^2.2.5",
-    "react": "^15.4.2",
-    "react-dom": "^15.4.2",
     "terra-mixins": "^1.0.0"
   },
   "scripts": {
-    "build": "babel src --out-dir lib",
-    "deploy": "build-storybook && gh-pages -d storybook-static",
-    "lint": "yarn lint-js && yarn lint-scss",
-    "lint-js": "eslint --ext .js,.jsx .",
-    "lint-scss": "stylelint src/**/*.scss",
-    "pretest": "yarn lint",
-    "release:major": "yarn test && yarn build && npm version major -m \"Released version %s\" && npm publish && git push --follow-tags",
-    "release:minor": "yarn test && yarn build && npm version minor -m \"Released version %s\" && npm publish && git push --follow-tags",
-    "release:patch": "yarn test && yarn build && npm version patch -m \"Released version %s\" && npm publish && git push --follow-tags",
-    "start": "start-storybook -p 8080",
-    "test": "jest",
-    "test:watch": "jest --watch",
-    "test:coverage": "jest --coverage"
+    "compile": "npm run compile:clean && npm run compile:build",
+    "compile:clean": "rm -rf lib",
+    "compile:build": "$(cd ..; npm bin)/babel src --out-dir lib --copy-files",
+    "lint": "npm run lint:js && npm run lint:scss",
+    "lint:js": "$(cd ..; npm bin)/eslint --ext .js,.jsx . --ignore-path ../../.eslintignore",
+    "lint:scss": "$(cd ..; npm bin)/stylelint src/**/*.scss",
+    "pretest": "npm run lint",
+    "release:major": "npm run test && node ../../scripts/release/release.js major",
```

Comment:
Small nit on the release scripts. You can remove `run` before `test` and call the script as `npm test && node ..` etc.

We've done in the other package.json files.
Both `npm start`[1](https://docs.npmjs.com/cli/start) and `npm test`[2](https://docs.npmjs.com/cli/test) don't need to specify `run` when called. Not entirely sure why.

I'm a 1/10 on this, as long as we are consistent on whatever we go forward with on all the package.json files.